Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM golang:1.19 AS builder
+FROM --platform=$BUILDPLATFORM golang:1.19 AS builder
 
 WORKDIR $GOPATH/src/github.com/aws/amazon-eks-pod-identity-webhook
 COPY . ./
-RUN GOPROXY=direct CGO_ENABLED=0 GOOS=linux go build -o /webhook -v -a -installsuffix nocgo -ldflags="-buildid='' -w -s" .
+ARG TARGETOS TARGETARCH
+RUN GOPROXY=direct CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -o /webhook -v -a -installsuffix nocgo -ldflags="-buildid='' -w -s" .
```

Comment:
What is the difference between using nocgo and `CGO_ENABLED=0`?  Not sure if I've seen nocgo before.

Reply:
I don't think it's needed anymore.